eng/Versions.props

```diff
@@ -84,7 +92,7 @@
     <MicrosoftCodeAnalysisCommonPackageVersion>3.0.0-beta4-final</MicrosoftCodeAnalysisCommonPackageVersion>
     <MicrosoftCodeAnalysisCSharpPackageVersion>3.0.0-beta4-final</MicrosoftCodeAnalysisCSharpPackageVersion>
     <MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>3.0.0-beta4-final</MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
-    <MicrosoftNETCoreApp30PackageVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreApp30PackageVersion>
+    <MicrosoftNETCoreApp30PackageVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreApp30PackageVersion>
```

Comment:
Interesting...it looks like this variable is only used for test apps, like https://github.com/aspnet/AspNetCore-Tooling/blob/17a95db34fdfb3afa04ac42a37b68183f667f710/src/Razor/test/testapps/Directory.Build.targets#L4-L8. If this variable is actually doing something, I'm surprised it still works. We stopped using RuntimeFrameworkVersion for testing in AspNetCore and replaced it with the `<KnownFrameworkReference Update...` syntax instead. RuntimeFrameworkVersion only impacts the runtime verisons used, not the targeting pack version (i.e. Microsoft.NETCore.App.Ref.)

Reply:
So should I update this PR to start using KnownFrameworkReference?

Reply:
I'll assume I don't need to for now. I can make a separate PR if we are concerned about it.